routes/login/login.controller.js

```diff
@@ -133,3 +235,135 @@ const postLogin = async (req, res, next) => {
 
   next()
 }
+
+const postAge = (req, res, next) => {
+  const ageYesNo = req.body.ageYesNo
+
+  req.session.login.ageYesNo = ageYesNo
+
+  next()
+}
+
+const postTaxableIncome = (req, res, next) => {
+  const taxableIncome = req.body.taxableIncome
+
+  req.session.login.taxableIncome = taxableIncome
+
+  if (taxableIncome !== 'Yes') {
+    return res.redirect('/offramp/taxable-income')
+  }
+
+  next()
+}
+
+const postResidenceScreening = (req, res, next) => {
+  const residenceScreening = req.body.residenceScreening 
+
+  req.session.login.residenceScreening  = residenceScreening 
+
+  if (residenceScreening  === 'No') {
+    return res.redirect('/offramp/residence')
+  }
+
+  next()
+}
+
+const postChildren = (req, res, next) => {
+  const children = req.body.children
+
+  req.session.login.children = children
+
+  if (children !== 'No') {
+    return res.redirect('/offramp/children')
+  }
+
+  next()
+}
+
+const postEligibleDependents = (req, res, next) => {
+  const eligibleDependents = req.body.eligibleDependents
+
+  req.session.login.eligibleDependents = eligibleDependents
+
+  if (eligibleDependents === 'No') {
+    return res.redirect('/eligibility/tuition')
+  }
+
+  if (eligibleDependents === 'Yes') {
+    return res.redirect('/eligibility/dependents-claim')
+  }
+
+  next()
+}
+
+const postEligibleDependentsClaim = (req, res, next) => {
+  const eligibleDependentsClaim = req.body.eligibleDependentsClaim
+
+  req.session.login.eligibleDependentsClaim = eligibleDependentsClaim
+
+  if (eligibleDependentsClaim === 'No') {
+    return res.redirect('/eligibility/tuition')
+  }
+
+  if (eligibleDependentsClaim === 'Yes') {
+    return res.redirect('/offramp/dependents')
+  }
+
+  next()
+}
+
+const postTuition = (req, res, next) => {
+  const tuition = req.body.tuition
+
+  req.session.login.tuition = tuition
+
+  if (tuition === 'No') {
+    return res.redirect('/eligibility/income-sources')
+  }
+
+  if (tuition === 'Yes') {
+    return res.redirect('/eligibility/tuition-claim')
+  }
+
+  next()
+}
```

Comment:
To extend upon what @katedee said earlier, we also have a `doYesNo` function to abstract some of this stuff (ie, lots of repetition in here), but doYesNo is build on the assumption that "yes" === go to a new question in this tree, whereas "no" means "eff this and go to the next topic". 

For these questions, sometimes the "right" answer is "yes" (eg, do you live in Ont), other times it is "no" (ie, do you have kids), so that basic assumption is broken.

I'll bring it up with Brian, but it kind of looks like your "custom" approach to this makes sense for the time being.